### ISSUE TITLE
How do you inline svgs?

### ISSUE THREAD

**Issue by marcofugaro** [pd4d10/vite-plugin-svgr#85]
The advantage of using svgr is that it inlines the .svg files in the react code. Any way to do this with `vite-plugin-svgr`?

Currently even the small files are not included:
<img width="881" alt="image" src="https://github.com/pd4d10/vite-plugin-svgr/assets/7217420/24d483eb-3f1a-4545-ad2b-681182a62e22">

**Comment by Nick-Nice** [comment]
I also want to solve this problem because there are too many svg in the project, and the size of the picture is very small but it will take up the http request. I hope the author can provide an inline solution.

**Comment by mattsputnikdigital** [comment]
I believe they are inlined but vite also emits them into the build folder. Are you seeing them being requested?

**Comment by marcofugaro** [comment]
@mattsputnikdigital yeah you're right, I don't see the HTTP requests to the .svg files, looks like they are actually inlined for me.

**Comment by pd4d10** [comment]
See https://github.com/pd4d10/vite-plugin-svgr/pull/71#issuecomment-1727006445

**Comment by allicanseenow** [comment]
Does this mean we can't inline svgs?

**Comment by allicanseenow** [comment]
Seems the online option is to set `assetsInlineLimit: 0` in `vite.config.js` if I want to line an svg inside url
